README.md

```diff
@@ -26,3 +25,9 @@ The "Roboto" font is included for development purposes, but is not included in t
 ## Get help
 
 -   If you have a general question, or are in need of support, please open a [Question issue](https://github.com/Lundalogik/lime-elements/issues/new?template=03_question.md) on GitHub.
+
+# Contributing
+
+-   To build and run the documentation locally on your machine, run `npm start`.
+-   To see what other scripts are available, run `npm run`.
+-   Please read our [guidelines for contributers](https://github.com/Lundalogik/lime-elements/CONTRIBUTING.md)
```

Comment:
ah nice. Maybe we should add it here? https://lundalogik.github.io/lime-elements/versions/next/#/Home/contributing.md/

Reply:
The `npm run` and `npm run start` you mean? Yeah, I guess. I think there are lots of things that should be updated in that file though… but there are more important things to spend our time on at the moment, imo.